docs/mddocs/DockerGuides/vllm_docker_quickstart.md

```diff
@@ -39,108 +40,452 @@ After the container is booted, you could get into the container through `docker
+    "created": 1727273935,
+    "model": "llama2-7b-chat",
+    "choices": [
+        {
+            "index": 0,
+            "text": "city that is known for its iconic landmarks, vibrant culture, and diverse neighborhoods. Here are some of the top things to do in San Francisco:. Visit Alcatraz Island: Take a ferry to the infamous former prison and experience the history of Alcatraz Island.2. Explore Golden Gate Park: This sprawling urban park is home to several museums, gardens, and the famous Japanese Tea Garden.3. Walk or Bike the Golden Gate Bridge: Take in the stunning views of the San Francisco Bay and the bridge from various v",
+            "logprobs": null,
+            "finish_reason": "length",
+            "stop_reason": null
+        }
+    ],
+    "usage": {
+        "prompt_tokens": 5,
+        "total_tokens": 133,
+        "completion_tokens": 128
+    }
+}
+```
 
 #### Multi-card serving
 
-vLLM supports to utilize multiple cards through tensor parallel. 
+For larger models (greater than 10b), we need to use multiple graphics cards for deployment. In the above script(`/llm/start-vllm-service.sh`), we need to make some modifications to achieve multi-card serving.
 
-You can refer to this [documentation](../Quickstart/vLLM_quickstart.md#4-about-tensor-parallel) on how to utilize the `tensor-parallel` feature and start the service.
+1. **Tensor Parallel Serving**: need modify the `-tensor-parallel-size` num, for example, using 2 cards for tp serving, add following parameter:
 
-#### Verify
-After the service has been booted successfully, you can send a test request using `curl`. Here, `YOUR_MODEL` should be set equal to `served_model_name` in your booting script, e.g. `Qwen1.5`.
+```bash
+--tensor-parallel-size 2
+```
 
+or shortening:
 
 ```bash
-curl http://localhost:8000/v1/completions \
--H "Content-Type: application/json" \
--d '{
-  "model": "YOUR_MODEL",
-  "prompt": "San Francisco is a",
-  "max_tokens": 128,
-  "temperature": 0
-}' | jq '.choices[0].text'
+-tp 2
+```
+
+2. **Pipeline Parallel Serving**: need modify the `-pipeline-parallel-size` num, for example, using 2 cards for pp serving, add following parameter:
+
+```bash
+--pipeline-parallel-size 2
+```
+
+or shortening:
+
+```bash
+-pp 2
+```
+
+3. **TP+PP Serving**: using tensor-parallel and pipline-parallel mixed, for example, using 2 cards for tp and 2 cards for pp serving, add following parameter:
+
+```bash
+--pipeline-parallel-size 2 \
+--tensor-parallel-size 2
+```
+
+or shortening:
+
+```bash
+-pp 2 \
+-tp 2
+```
+
+You can refer to this [documentation](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Quickstart/vLLM_quickstart.html#about-tensor-parallel) for more information on how to utilize the `tensor-parallel` feature and start the service.
+
+### Quantization
+
+The accuracy of the quantitative model is reduced from FP16 to INT4, which effectively reduces the file size by about 70 %. The main advantage is lower delay and memory usage.
```

Comment:
`reduces the file size by about 70 %` is about the model file size? In vLLM.entrypoint (LLM, openai.api_server, etc.) we are still using the origin model file instead of the ipex_llm converted file, the quantization convert occurs in CPU memory.